recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
 {% set name = "proto-plus" %}
-{% set version = "1.19.9" %}
-
+{% set version = "1.22.1" %}
+{% set sha256 = "6c7dfd122dfef8019ff654746be4f5b1d9c80bba787fe9611b508dd88be3a2fa" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/proto-plus-{{ version }}.tar.gz
-  sha256: 4ca4055f7c5c1a2239ac7a12770a76a16269f58d3f01631523c20fc81dbb14a7
+  sha256: {{ sha256 }}
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
+  script: python -m pip install . -vv --no-deps
```

Comment:
I guess `{{ PYTHON }}` is better than `python`.

Reply:
I see both idioms out there, but I'm not convinced. If conda build is working properly, then `python` should resolve to the proper path.